cmake/projects/Lua/hunter.cmake

```diff
@@ -0,0 +1,23 @@
+if(DEFINED HUNTER_CMAKE_PROJECTS_LUA_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_LUA_HUNTER_CMAKE_ 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Lua
+    VERSION
+    "5.3.2"
+    URL
+    "https://github.com/hunter-packages/lua/archive/5.3.2.tar.gz"
+    SHA1
+    5395f8637c14b41f950b28f2c1dba65f30fa7030
+)
+
```

Comment:
Probably `hunter_configuration_types(Lua CONFIGURATION_TYPES Release)` need to be added here. By default debug + release variant is built but executable has same name for both (i.e. will be overwritten).

See wiki: https://github.com/ruslo/hunter/wiki/usr.adding.new.package#build-types